src/main/java/pl/scoutCamp/api/controller/rest/LoginController.java

```diff
@@ -8,24 +8,27 @@
 import org.springframework.security.core.Authentication;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-@RestController
-@AllArgsConstructor
-public class LoginController {
+//@RestController
```

Comment:
Should this code be commented? If yes please add some TODO/FIXME and explain what should be done there.